releases/point-2022-4-1-release-note.md

```diff
@@ -0,0 +1,69 @@
+---
+description: This article lists new features, improvements, and bug fixes in SysKit Point version 2022.4.1.
+--- 
+
+# SysKit Point 2022.4.1
+
+[Schedule a demo](https://www.syskit.com/products/point/request-a-demo/) and [tell us what you think.](https://www.syskit.com/company/contact-us/)
+
+**Product version:** 2022.4.1
+
+**Build number:** 4
```

Comment:
ovo budemo promijenili kad saznamo koji točno bude build number kada se napravi release branch.

Reply:
build numbner je 1